apm-sniffer/apm-toolkit-activation/apm-toolkit-logging-common/src/main/resources/META-INF/services/org.apache.skywalking.apm.agent.core.boot.BootService

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (C) 2021 Baidu, Inc. All Rights Reserved.
```

Comment:
please remove it

Reply:
@kezhenxu94 This is a hit, why the SkyWalking eye doesn't detect this?

Reply:
> @kezhenxu94 This is a hit, why the SkyWalking eye doesn't detect this?
 
The Apache license header is there correctly, but "Copyright (C) 2021 Baidu, Inc. All Rights Reserved." is an extra line, skywalking-eyes doesn't take this into consideration for now